src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -4025,7 +4025,7 @@ class Program
 }
 ref struct R<T> { }
 ";
-            var comp = CreateCompilation(new[] { source, UnscopedRefAttributeDefinition });
+            var comp = CreateCompilation(source);
```

Comment:
I didn't understand why we don't need to pass the attribute definition any more. Maybe because we're compiling against net7?

Reply:
> Maybe because we're compiling against net7?

That's correct.

Reply:
The attribute is not used in the test.